apps/full-site-editing/full-site-editing-plugin/readme.txt

```diff
@@ -1,9 +1,9 @@
 === Full Site Editing ===
-Contributors: automattic, get_dave, marekhrabe, mppfeiffer, obenland
+Contributors: alexislloyd, allancole, automattic, codebykat, copons, dmsnell, get_dave, glendaviesnz, helpingcat, iamtakashi, Joen, kwight, marekhrabe, mattwiebe, mmtr86, mppfeiffer, nrqsnchz, obenland, okenobi, vindl
```

Comment:
Thanks @obenland Do you mind updating this to `gwwar` from `helpingcat`? I reserved both, but I figure it might be easier to sync my github and dotorg handles.